indent/vue.vim

```diff
@@ -9,7 +9,13 @@ endif
 
 function! s:get_indentexpr(language)
   unlet! b:did_indent
-  execute 'runtime! indent/' . a:language . '.vim'
+  let path = ""
+  if exists('g:vim_vue_indent_paths') && has_key(g:vim_vue_indent_paths, a:language)
+    let path = g:vim_vue_indent_paths[a:language]
```

Comment:
Indent files from plugins are already loaded by default. Isn't this working for you?

Reply:
for me there were cases where you can have multiple plugins with indent rules.  For example, eclim and vim-javascript both contain javascript rules, however vim-javascript is far superior for me.  The old method always resulted in eclim loading for vue files. 

The other suggested fixes for using set ft= html.javascript or vice-versa worked, but also broke completion and other plugins.

Reply:
also I think the old code was not actually calling the indent function unless I am missing something..I don't know vimscript that well, but seemed that way to me